producer/writer/consumer_writer.go

```diff
@@ -331,7 +330,7 @@ func (w *consumerWriterImpl) connectWithRetry(addr string) (io.ReadWriteCloser,
 		w.continueFn,
 		fn,
 	); attemptErr != nil {
-		return nil, fmt.Errorf("failed to connect to address %s, %v", addr, attemptErr)
+		return nil, attemptErr
```

Comment:
Is it not useful to have the address in the error?

Reply:
Yeah I'm already including the address where the error is logged, so it's a bit redundant.